app/Models/SCIMUser.php

```diff
@@ -9,8 +9,7 @@ class SCIMUser extends User
     protected $throwValidationExceptions = true; // we want model-level validation to fully THROW, not just return false
 
     public function __construct(array $attributes = []) {
-        $attributes['password'] = "*NO PASSWORD*";
-        // $attributes['activated'] = 1;
         parent::__construct($attributes);
+        $this->noPassword();
```

Comment:
I think this should be assigning to `$attributes['password']`?

Reply:
So, `SCIMUser` extends `User` - so it has access to the same `->noPassword()` method. The `__construct()` method is only invoked on `new Blah()` so the order of the `parent::__construct()` method going first, or the `$this->noPassword()` method _shouldn't_ matter so much, I think?

Reply:
(in other words, it's not calling `create()` which is where the ordering would matter more)

Reply:
`noPassword()` returns a string so calling the method without assigning it to something doesn't do anything though right?

Reply:
Oh, d'oh, you're right. I have a different proposal I want to make, which I will do separately from this though.